packages/connect-playwright/package.json

```diff
@@ -36,10 +36,11 @@
     "license-header": "license-header"
   },
   "peerDependencies": {
-    "@connectrpc/connect": "^0.13.2",
+    "@connectrpc/connect": "^1.0.0-rc1",
     "@playwright/test": "^1.0.0"
   },
   "devDependencies": {
-    "@playwright/test": "^1.37.1"
+    "@playwright/test": "^1.37.1",
+    "@types/node": "^20.6.0"
```

Comment:
Curious, but I guess that must be related to changes in playwright, or something odd in the lock file...